src/sync.rs

```diff
@@ -0,0 +1,9 @@
+//! Loom needs its own syncronization types to be used in order to work
```

Comment:
As this is just plumbing code, I think an inline module in lib.rs would do just fine.